### PR TITLE
Py3 Support

### PR DESCRIPTION
**Pull request by nickumia-reisys**
This PR is to support the py3 upgrade found [here](https://github.com/GSA/datagov-ckan-multi/issues/569)

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install requirements (py2)
+      if: ${{ matrix.ckan-version == '2.8' }}
+      run: |
+        pip install -r requirements-py2.txt
+        pip install -r dev-requirements-py2.txt
+        pip install -e .
+    - name: Install requirements (py3)
```
Comment (jbrown-xentity) [inline .github/workflows/test.yml:60]: I would consider merging these scripts since they rely on the same check (ckan >= 2.9)

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,30 @@
+CKAN_VERSION ?= 2.8
+COMPOSE_FILE ?= docker-compose.yml
+COMPOSE_2_8_FILE ?= docker-compose.2.8.yml
```
Comment (jbrown-xentity) [inline Makefile:3]: Can we remove this?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+    cd ckanext-datagovcatalog
+    python setup.py develop
+    pip install -r requirements.txt
+    pip install -r dev-requirements.txt
+
+**Note: use the py2-equivalents of the requirement files if running on python 2.
+
+# Running the Tests
+
+## Tests
```
Comment (jbrown-xentity) [inline README.md:84]: Remove as unnecessary

File: ckanext/datagovcatalog/tests/test_notifications.py [hunk 4]
```diff
@@ -86,7 +88,10 @@ def test_create_harvest_source_with_no_org(self):
         new_rec_action = toolkit.get_action("harvest_get_notifications_recipients")
         new_recipients = new_rec_action(context, {'source_id': source_id})
 
-        assert_in({'name': u'test.ckan.net', 'email': None}, new_recipients)
+        # For Local testing,
+        # assert {'name': u'default', 'email': None} in new_recipients
+        # For Github Actions,
+        assert {'name': u'test.ckan.net', 'email': None} in new_recipients
```
Comment (jbrown-xentity) [inline ckanext/datagovcatalog/tests/test_notifications.py:94]: Can you describe why this is necessary?

File: requirements.txt [hunk 5]
```diff
@@ -0,0 +1,6 @@
+-e git+https://github.com/nickumia-reisys/ckanext-harvest.git@9d1f647d247c16b6c3acba26e321e9500cafb18c#egg=ckanext-harvest
```
Comment (jbrown-xentity) [inline requirements.txt:1]: Can you describe why we have to be on a personal git commit? Should we merge this to GSA fork?

**Review by jbrown-xentity — APPROVED** [review]
Looks good; a few clarifying comments and we should be good to go!

File: .github/workflows/test.yml (hunk 1 above)
Comment (nickumia-reisys) [reply .github/workflows/test.yml]: Done.

File: Makefile (hunk 2 above)
Comment (nickumia-reisys) [reply Makefile:3]: Done.

File: README.md (hunk 3 above)
Comment (nickumia-reisys) [reply README.md:84]: I don't think there's anything to remove.

File: ckanext/datagovcatalog/tests/test_notifications.py (hunk 4 above)
Comment (nickumia-reisys) [reply ckanext/datagovcatalog/tests/test_notifications.py:94]: Fixed.

**Pull request merged**